run_toshi_hazard_post_script.py

```diff
@@ -120,15 +121,21 @@
             ["Subduction Interface"],
             ["Subduction Intraslab"],
         ],
-        which_interfaces=["only_HIK", "only_PUY", "HIK_and_PUY"],
+        which_interfaces=[SubductionInterface.HIK, SubductionInterface.PUY, SubductionInterface.HIK|SubductionInterface.PUY],
     )
 )
 
 ### concatenate the logic_tree_pair_lists
+# logic_tree_pair_list = (
+#     logic_tree_pair_list1 + logic_tree_pair_list2 + logic_tree_pair_list3
+# )
```

Comment:
Temporary code?

Reply:
sorry, this was all commented out for debugging. Uncommented now.